foreshadow/transformers/transformers.py

```diff
@@ -268,7 +403,7 @@ def transform(self, X, y=None):
             :obj:`numpy.ndarray`: Empty numpy array
 
         """
-        return np.array([])
+        return pd.DataFrame([])
```

Comment:
This is smart, so that we don't have to manually wrap the _Empty, good catch :)

Reply:
:) thanks